src/objects/structure.rs

```diff
@@ -130,7 +130,6 @@ impl Structure {
             Structure::Observer(v) => Some(v),
             Structure::PowerBank(v) => Some(v),
             Structure::PowerSpawn(v) => Some(v),
-            Structure::Portal(v) => Some(v),
             Structure::Rampart(v) => Some(v),
```

Comment:
That looks like a non-comprehensive pattern matching.

Reply:
I thought I'd actually compiled this code...